openmetadata-ui/src/main/resources/ui/src/components/GlobalSetting/GlobalSetting.less

```diff
@@ -11,30 +11,62 @@
  *  limitations under the License.
  */
 
-.global-setting-left-panel.ant-menu-inline {
-  box-shadow: 0px 2px 8px 0px rgba(0, 0, 0, 0.15);
-  width: 290px;
-  padding: 16px 0;
+.global-setting-left-panel.ant-menu-root.ant-menu-inline {
+  background: #ffffff;
+  border: 1px solid rgba(229, 231, 235, var(--tw-border-opacity));
```

Comment:
Do we need an opacity variable here?

Reply:
I have given this as per the mocks. Should I remove it?

Reply:
No, Can we use less variables then?

Reply:
were did you declare `--tw-border-opacity` variable? this is the tailwind style of CSS, I think we should directly provide number

Reply:
Done